listenbrainz/webserver/static/js/src/UserArtists.test.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from "react";
+import { mount, shallow } from "enzyme";
+
+import UserArtists from "./UserArtists";
+import * as userArtistsResponse from "./__mocks__/userArtists.json";
+import * as userArtistsProcessDataOutput from "./__mocks__/userArtistsProcessData.json";
+
+const props = {
+  user: {
+    id: 0,
+    name: "dummyUser",
+  },
+  apiUrl: "apiUrl",
+};
+
+describe("User Artists Page", () => {
+  it("renders", () => {
+    // We don't need to call componentDidMount during "mount" because we are
+    // passing the data manually, so mock the implementation once.
+    jest
+      .spyOn(UserArtists.prototype, "componentDidMount")
+      .mockImplementationOnce((): any => {});
+
+    const wrapper = mount<UserArtists>(<UserArtists {...props} />);
+    const instance = wrapper.instance();
+
+    // eslint-disable-next-line dot-notation
```

Comment:
I don't like so many `eslint-disable-next-line`s, can we just make these things public?